src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -275,7 +274,6 @@ public static ExtensionList<StepDescriptor> all() {
      */
     public static Iterable<StepDescriptor> allMeta() {
         return Iterables.filter(all(), new Predicate<StepDescriptor>() {
-            @SuppressFBWarnings(value="NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE", justification="all() will not return nulls")
```

Comment:
SpotBugs still seems to require this; can check whether a parent POM update fixes that…